packages/fxa-content-server/app/scripts/lib/glean/index.ts

```diff
@@ -168,6 +188,36 @@ const recordEventMetric = (eventName: string) => {
     case 'login_totp_code_success_view':
       login.totpCodeSuccessView.record();
       break;
+    case 'password_reset_create_new_submit':
+      passwordReset.createNewSubmit.record();
+      break;
+    case 'password_reset_create_new_success_view':
+      passwordReset.createNewSuccessView.record();
+      break;
+    case 'password_reset_create_new_view':
+      passwordReset.createNewView.record();
+      break;
+    case 'password_reset_recovery_key_create_new_submit':
+      passwordReset.recoveryKeyCreateNewSubmit.record();
+      break;
+    case 'password_reset_recovery_key_create_new_view':
+      passwordReset.recoveryKeyCreateNewView.record();
+      break;
+    case 'password_reset_recovery_key_create_success_view':
+      passwordReset.recoveryKeyCreateSuccessView.record();
+      break;
+    case 'password_reset_recovery_key_submit':
+      passwordReset.recoveryKeySubmit.record();
+      break;
+    case 'password_reset_recovery_key_view':
+      passwordReset.recoveryKeyView.record();
+      break;
+    case 'password_reset_submit':
+      passwordReset.submit.record();
+      break;
+    case 'password_reset_view':
+      passwordReset.view.record();
+      break;
```

Comment:
Thank you for adding these.  The Backbone UI is not longer used to handle password resets so these won't be used.  But it doesn't hurt anything to leave them here.